tests/integration/tier0/check-internet-connection/main.fmf

```diff
@@ -1,6 +1,20 @@
-summary: internet connection checks
+summary: Internet connection checks
+description: |
+    Verify, that internet connection check works as expected for both available and unavailable connection.
 
 tier: 0
 
-test: |
-  pytest -svv
+tag+:
+    - internet-connection
+
+/available_connection:
+    tag+:
+        - available_connection
+    test: |
+        pytest -svv -m available_connection
+
+/unavailable_connection:
+    tag+:
+        - unavailable-connection
+    test: |
+        pytest -svv -m unavailable_connection
```

Comment:
I have modified the description and the metadata to align with proposed changes for tier0 refactoring.
Hope you don't mind @r0x0d :)

Reply:
That's fine by me